vars/managerPipeline.groovy

```diff
@@ -238,10 +247,12 @@ def call(Map pipelineParams) {
                                             export SCT_SCYLLA_VERSION="${params.scylla_version}"
                                         elif [[ ! -z "${params.gce_image_db}" ]] ; then
                                             export SCT_GCE_IMAGE_DB="${params.gce_image_db}"  #TODO: remove it once scylla_version supports gce image detection
+                                        elif [[ ! -z "${params.azure_image_db}" ]] ; then
+                                            export SCT_AZURE_IMAGE_DB="${params.azure_image_db}"  #TODO: remove it once scylla_version supports azure image detection
```

Comment:
scylla_version supports azure image detection. See [`sct_config.py` ](https://github.com/scylladb/scylla-cluster-tests/blob/master/sdcm/sct_config.py#L1527)
Do we need that then?

Reply:
I tried to use scylla_version=5.0, and the image that was found is:
`/subscriptions/6c268694-47ab-43ab-b306-3c5514bc4112/resourceGroups/SCYLLA-IMAGES/providers/Microsoft.Compute/images/ScyllaDB-5.0.dev-0.20220125.eb42213db4d-1-build-260`
More than a half year ago

Reply:
So I don't think I can count on it for now. I will amend it in the future

Reply:
yes, there was some problem with tagging Azure images.

Reply:
@ShlomiBalalis I verified it today and it looks it is fixed now. Listing images for 5.0 shows recent images properly.